src/OSPSuite.Core/Services/IHeavyWorkManager.cs

```diff
@@ -19,15 +20,15 @@ public interface IHeavyWorkManager
       ///    action was canceled
       /// </summary>
       /// <param name="heavyWorkAction">Action that need to be performed</param>
-      bool Start(Action heavyWorkAction);
+      bool Start(Action heavyWorkAction, CancellationTokenSource cts = default);
```

Comment:
Given that the other param is described in the XML comment the new one has to be described too otherwise it's a compiler warning